superbench/config/azure_hbv3.yaml

```diff
@@ -0,0 +1,85 @@
+# SuperBench Config
```

Comment:
It's great, can we separate PRs for these 2 configurations? e.g. Support Azure HBv3/HBv4 SKU. It would be easy to list the new features for Azure HBv3/HBv4 support for release.

Reply:
addressed